tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/UpToDate/BuildUpToDateCheckTests.cs

```diff
@@ -1506,8 +1506,8 @@ public async Task IsUpToDateAsync_True_Kinds_InputNewerThanOutput_NoKindIgnored(
                 lastSuccessfulBuildStartTimeUtc: lastBuildTime,
                 lastItemsChangedAtUtc: itemChangeTime);
 
-            _fileSystem.AddFile(@"C:\Dev\Solution\Project\Input",        inputTime);
-            _fileSystem.AddFile(@"C:\Dev\Solution\Project\TaggedInput",  inputTime);
+            _fileSystem.AddFile(@"C:\Dev\Solution\Project\Input",        input2Time);
+            _fileSystem.AddFile(@"C:\Dev\Solution\Project\TaggedInput",  input1Time);
```

Comment:
I created different timestamps for these values to break any race condition where values were identical.